src/CoreBundle/Resources/views/snippets-cms/CMSDocumentManager/standard.html.twig

```diff
@@ -7,54 +7,82 @@
     var _transImagePixel = '{{'chameleon_system_core.text.pixel'|trans}}';
 </script>
 
-<div class="row" style="height: 100%; margin-right: 0px;">
-    <div class="col-md-3" style="height: 100%;">
-        <h1>
-            {{'chameleon_system_core.cms_module_media_manager.folder'|trans}}
-        </h1>
-        <div id="treePlacer"></div>
-    </div>
-
-    <div style="height: 100%; padding-right: 0px" class="col-md-9">
-        <div class="btn-group">
-            {% if wysiwygMode != '' %}
-                <button type="button" class="btn btn-sm btn-primary" onclick="InsertDocumentsInWYSIWYG();"><span class="btn-icon" style="background-image: url({{sPathTheme}}/images/icons/attach.png);">{{'chameleon_system_core.document_manager.insert_into_text'|trans}}</span></button>
-            {% endif %}
-
-            {% if recordID != '' %}
-                <button type="button" class="btn btn-sm btn-primary" onclick="chooseSelectedFiles();"><span class="btn-icon" style="background-image: url({{sPathTheme}}/images/icons/accept.png);">{{'chameleon_system_core.document_manager.action_select'|trans}}</span></button>
+<div{% if displayFull is not defined or displayFull == false %} class="p-4"{% endif %}>
+    <div class="row" style="height: 100%; margin-right: 0px;">
+        <div class="col-12 col-xl-3 mb-4" style="height: 100%;">
+            {% if displayFull is not defined or displayFull == false %}
+                <h3>
+                    {{'chameleon_system_core.cms_module_media_manager.folder'|trans}}
```

Comment:
The translation key is from the media manager. Please add a copy for the document manager.

Reply:
ok, done